R/pkg/NAMESPACE

```diff
@@ -245,6 +245,7 @@ exportMethods("%<=>%",
               "current_date",
               "current_timestamp",
               "hash",
+              "xxhash64",
```

Comment:
this is sorted (or should be, except obvious problem with "hash")

Reply:
Ah. Fixed here and in `generics.R`.